src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -27,6 +27,8 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
   }
 
   @property() private _config?: MarkdownCardConfig;
+  @property() private content?: string = "";
```

Comment:
Prefix private properties with `_`

```suggestion
  @property() private _content: string = "";
```